### DISCUSSION TITLE
We are tinyBlogPub, now!!!

### DISCUSSION THREAD

**Discussion by nigini** [tinyBlogPub/microblog.pub#25] [Announcements]
Hey, y'all.
Again, thanks for the input on the [name change](#10). We are now https://tinyblog.pub/ :)

**Reminder:** 
  1. Although GitHub redirects the previous repo names to the new ones, it is recommended to change them ASAP.
  2. Also, let's go back to previous blog posts and change the URLs
  3. Please contribute changes to the ORG front page!

Now, I guess it is time to dream about what we will do with it:
- a simple blog at the core (something like [this](https://flask-blogging.readthedocs.io/)?)
- modularize and integrate implementation of [MicroPub](https://micropub.rocks/)
- modularize and integrate implementation of [ActivityPub](https://activitypub.rocks/) (I'd love to maintain a reliable Python server.)

What do you dream of working on?

**Comment by akohlsmith** [comment]
This is fantastic news, congrats! I keep hoping for a standard Mastodon API (even minimal) so I can use existing mobile clients (I do not like even "mobile friendly" HTML) -- I see you've been working on modularizing/compartmentalizing the ActivityPub and MicroPub subsystems, so I have hope that a minimal client-server API may get exposed at some point. :-)
